recipes/tdameritrade/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tdameritrade" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 692baf738dc8c5f9250764ac7db101171c5df8dd59e39bae77d2a7ff41a02479
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - tdameritrade-auth = tdameritrade.auth:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - ipython >=7.0.1
+    - pandas >=0.22.0
+    - requests >=2.22.0
+    - selenium >=3.141.0
+    - ujson >=1.35
+
+test:
+  requires:
+    - flake8 >=3.7.8
+    - pytest >=4.3.0
+  imports:
+    - tdameritrade
+  commands:
+    - flake8 tdameritrade
```

Comment:
I would say that we don't need to run `flake8` as a test for conda-forge. For the `pytest` bit that's causing the error, you need a `source_files` section (https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-files) that includes `tdameritrade/tests`.

Also with regard to my previous comment about `numpy`, if you don't import it in the project, then yes, you can leave it off of the dependencies list.